app/templating/schema_context.py

```diff
@@ -56,3 +59,7 @@ def _build_respondent(metadata):
         "trad_as": metadata["trad_as"],
         "trad_as_or_ru_name": metadata["trad_as"] or metadata["ru_name"],
     }
+
+
+def json_safe(data):
+    return data.replace('\\', r'\\').replace('"', r'\"')
```

Comment:
Yeah fully understand. I'm not satisfied with the solution but cannot thing of any other way without rewriting how we do string injection. Regarding the `json.dumps` we would have to do `answer = json.dumps(answer)[1:-1] if answer else answer` as it adds in extra quotes